docs/manual/docs/cmd/spo/folder/folder-move.md

```diff
@@ -0,0 +1,55 @@
+# spo folder move
```

Comment:
A reference to the .md file should be included in mkdocs.yml or the file won't be included in the generated documentation.